packages/core/cache/src/FSCache.js

```diff
@@ -141,12 +132,12 @@ export class FSCache implements Cache {
             this.#getFilePath(key, i),
             typeof contents === 'string'
               ? contents.slice(
-                  i * this.#writeLimitChunk,
-                  (i + 1) * this.#writeLimitChunk,
+                  i * WRITE_LIMIT_CHUNK,
+                  (i + 1) * WRITE_LIMIT_CHUNK,
                 )
               : contents.subarray(
-                  i * this.#writeLimitChunk,
-                  (i + 1) * this.#writeLimitChunk,
+                  i * WRITE_LIMIT_CHUNK,
+                  (i + 1) * WRITE_LIMIT_CHUNK,
```

Comment:
Did we want to revert the chunking here too seeing as that was the potential issue you were concerned about?